Add instructions for DATADOG_SITE (#11477)
DataDog/documentation: content/en/serverless/installation/nodejs.md

### Configure credentials

For a quick start, configure Datadog and [AWS credentials][1] using the following command. For production applications, consider supplying the environment variables or credentials in a more secure manner.

```bash
export DATADOG_API_KEY="<DD_API_KEY>"

> export DATADOG_SITE="<DD_SITE>" # eg datadoghq.com, datadoghq.eu, etc

Reviewer: ```suggestion
export DATADOG_SITE="<DD_SITE>" # such as datadoghq.com or datadoghq.eu
```

Reviewer 2: @DarcyRaynerDD let's list out all the possible values: `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. It's sometimes very difficult for the customer to figure out the right full site name, since they are not named very consistently.

>   - Set `DD_LAMBDA_HANDLER` to your original handler, for example, `myfunc.handler`.

Reviewer 3: Do we want to add DATADOG_SITE here as well?

Author: I think this section deals with the command line parameters, not env vars.
